app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    unique = set()
+
+    def inner(*args):
+        if args not in unique:
+            unique.add(args)
+            print("Calculating new result")
+            return func(*args)
+        else:
+            print("Getting from cache")
+            return unique
```

Comment:
Function should return result of executing `func` with given arguments, but it returns `unique` instead

Reply:
This result should be stored after first execution with such args